core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -10040,6 +10040,86 @@ class KafkaApisTest extends Logging {
     assertEquals("Use of Kafka internal topics __consumer_offsets,__transaction_state,__share_group_state in a Kafka Streams topology is prohibited.", response.data.errorMessage())
   }
 
+  @Test
+  def testStreamsGroupHeartbeatRequestMissingTopics(): Unit = {
```

Comment:
This name is a bit misleading. I thought an exception should be thrown verified in this test. What about `testStreamsGroupHeartbeatRequestWithInternalTopicsToCreate()`

Please also rename the test below accordingly.